### ISSUE TITLE
Accept domain.tld param for --force option

### ISSUE THREAD

**Issue by nickpearson** [lukas2511/letsencrypt.sh#185]
Thanks for the awesome library. I'd like to be able to force-renew just one cert from the `domains.txt` file. Is it possible to add an optional parameter to the `--force` option to accomplish this?

For example, if `domains.txt` contains:
```
example.com www.example.com m.example.com
test.net www.test.net
```
it would be nice if `./letsencrypt.sh --force test.net` would cause only the `test.net` cert to renew (assuming the `example.com` cert doesn't otherwise need to be renewed).

If an optional parameter isn't possible, perhaps a new option like `--force-domain`?

I'm not an expert with bash but I'd be happy to give this a try and submit a PR if this is something you'd consider merging.

**Comment by lukas2511** [comment]
Current way to do this would be `letsencrypt.sh -c -d test.net -d www.test.net -x`.

Since I see this as more like an edge-case (you'd basically only need to do this in case you lost a private key) I don't think I'll add another option to make this easier.

**Comment by nickpearson** [comment]
I understand, and thanks for showing the proper usage of the `-d` option.

I ended up solving this before your reply by just deleting the `certs/domain.tld` directory for the cert I needed to force-renew and then running `letsencrypt.sh -c` again. (I needed to renew a certificate that was issued before Let's Encrypt fixed the issue that was causing certs not to work on any browser in Windows XP.) And you're right, this was an edge case.
